packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.tsx

```diff
@@ -72,11 +77,16 @@ const LanguageEditor = ({ selectedLanguageTag, onFormStateChange }: LanguageEdit
       .json<CustomPhraseResponse>();
 
     void mutate(updatedCustomPhrase);
+    onUpdated(updatedCustomPhrase);
     toast.success(t('general.saved'));
   });
 
   useEffect(() => {
-    reset(customPhrase?.translation ?? emptyUiTranslation);
+    reset(
+      customPhrase && Object.keys(customPhrase.translation).length > 0
```

Comment:
`Object.keys(customPhrase?.translation || {}).length > 0 ?`

Reply:
No, if we use the optional chaining operator here, we cannot infer the type of `customPhrase` in the coming `customPhrase.translation`.

Reply:
k not a big deal